### DISCUSSION TITLE
有没有萌新能帮上忙的部分

### DISCUSSION THREAD

**Discussion by ooPG233** [bangumi/frontend#30] [General]
rt，刚写了几个月前端，开发启动前的设计肯定插不上话了，好奇开始开发后会不会有可能帮上忙的部分

**Comment by Ayase-252** [comment]
欢迎各种形式的贡献，包括且不限于提升测试覆盖率、完善文档、修 typo 等

**Comment by Aloento** [comment]
我可以帮忙前端后端做设计，所以如果你需要的话可以告诉我你想让我写哪一部分，咱不太会自己在大项目中找活（

**Comment by trim21** [comment]
现在可以做这个 https://github.com/bangumi/frontend/issues/235
